src/main/integrations/companion-server/api/v1/index.ts

```diff
@@ -148,6 +149,22 @@ const CompanionServerAPIv1: FastifyPluginCallback<CompanionServerAPIv1Options> =
           break;
         }
 
+        case "changeSong": {
```

Comment:
A better name such as `changeVideo` might be suited for this as we don't use the term `song` anywhere and neither does YTM

Reply:
Done.